README.md

```diff
@@ -22,7 +22,9 @@ You can follow the [Suricata user guide](https://docs.suricata.io/en/latest/) to
 Contributing
 ------------
 
-We're happily taking patches and other contributions. Please see https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Contributing for how to get started.
```

Comment:
Running `git grep redmine.openinfosecfoundation.org/projects/suricata/wiki` shows there is also this link to change in .github/CONTRIBUTING.md` ;-)

Reply:
🙇🏽